docs/xdr/features/investigate/alerts.md

```diff
@@ -34,6 +34,71 @@ The urgency can have two different representations on the interface: a numerical
 | Major | [60-80[ |
 | Urgent | [80-100] |
 
+### Alert Similarity 
+Alert similarity (Occurence) is the process by which we collect similar events in the same alert. 
+The information is available in the Alerts table → Column `Occurrence`. 
+
+!!! note
+    Alert similarity has nothing to do with similar alerts. Check this [section](#similar-alerts) to learn more about similar alerts.
+
+**Example**
+
+If an alert has 24 occurrences, it means that it contains 24 events that were classified as similar and put in the same alert. 
+
+### Similarity strategies
+
```

Comment:
```suggestion
Alerts are considered similar if a list of fields defined by the similarity strategy have the same values for all events. Some fields may also be grouped together to specify that their values may be swapped.
```

I'm not entirely satisfied with this but can't find a clear and simple explanation :sweat_smile: